CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   * Emmet based abbreviation expansion is not yet supported (See note in readme for how to enable through your settings.json)
```

Comment:
```suggestion
  * Emmet based abbreviation expansion is not yet supported (See note in readme for how to enable through your settings.json)
  * Directive and TagHelper Attribute Completions are now committed contextually with `=`, `:` and ` ` commit characters. However, there's a known issue by which these characters will be added to the attribute value (`@att|` commit using `=` yields `@attributes="=|"` instead of `@attributes="|"` as intended). ([#22150](https://github.com/dotnet/aspnetcore/issues/22150))
```